itou/www/approvals_views/views.py

```diff
@@ -223,6 +224,7 @@ def declare_prolongation(request, approval_id, template_name="approvals/declare_
         "form": form,
         "preview": preview,
         "s3_upload": s3_upload,
+        "unfold_details": form.data.get("reason") in PROLONGATION_REPORT_FILE_REASONS,
```

Comment:
Est-ce que passer `PROLONGATION_REPORT_FILE_REASONS` et faire le `in` dans le template n'éviterais pas une indirection et permettrais de comprendre tout de suite dans quel cas on affiche ces champs ?

Reply:
Pas convaincu de tirer les notations métiers dans le template, même si le point de l'indirection est bon.